CMakeLists.txt

```diff
@@ -810,14 +822,9 @@ foreach(sourcefile ${C_TEST_EXES})
     add_test(NAME ${exename} COMMAND ${exename}${ARTIFACT_SUFFIX})
     add_dependencies(check ${exename}${ARTIFACT_SUFFIX})
 endforeach(sourcefile ${C_TEST_EXES})
-add_subdirectory(tools)
+endif()
 
-# Installation and packaging for Linux
-if(NOT WIN32)
```

Comment:
I am skeptical about creating a viable installation on windows. However, if you decided to go ahead I'd prefer to make it optional.